docs/17-porting-tips.md

```diff
@@ -153,6 +167,7 @@ they are still supported, but will be removed in the next release.
                                                        end
                                                        -- This will also un-minimize
                                                        -- the client, if needed
+                                                       `client.focus` = c
```

Comment:
I was just following the convention used in the lines preceding my changes.  I don't know why the backticks were used other than as a code highlight.